processing/data_collection/gazette/pipelines.py

```diff
@@ -22,6 +22,21 @@ def process_item(self, item, spider):
         return item
 
 
+class DefaultValuesPipeline:
+    """ Add defaults values field, if not already set in the item """
+
+    default_field_values = {
+        "territory_id": lambda item, spider: getattr(spider, "TERRITORY_ID"),
+        "scraped_at": lambda item, spider: dt.datetime.utcnow(),
+    }
+
+    def process_item(self, item, spider):
+        for field in self.default_values:
```

Comment:
**issue**: `self.default_values` is not defined. Isn't it `self.default_field_values`?